SingularityUI/app/utils.coffee

```diff
@@ -227,6 +227,14 @@ class Utils
         splits = taskId.split('-')
         splits[splits.length - 3]
 
+    @getDeployIdFromTaskId: (taskId) ->
```

Comment:
I think it might be a bit cleaner and more reusable to have a model for the TaskId. i.e. have something like a `fromString` method for the taskId model to set the attributes. Reusable where we need it, and you are then only calling the split once when you build the object instead of multiple times for the same string